pkg/log/log.go

```diff
@@ -46,23 +48,62 @@ func WithContext(ctx context.Context) {
 	Info().WithContext(ctx)
 }
 
-func (l *logger) Print(msg string) {
+func (l *logger) Print(msg string, fields field.Fields) {
```

Comment:
Please decouple accepting fields as a param and extracting the fields from the context.

Reply:
For now, let's have the `Print` signature be:
```suggestion
func (l *logger) Print(msg string) {
```
The `fields` argument is not immediately needed, right? They will be extracted from the context, no?
We can easily extend the `Print` function later to accept other arguments. However, we have not gone through the exercise of evaluating the tradeoffs of different alternatives.
We decided to use `WithContext()` and `WithError()` for now as a mean to achieve the immediate desired functionality.